### DISCUSSION TITLE
Not able to see certain types of errors

### DISCUSSION THREAD

**Discussion by saifobeidat** [quasarframework/quasar#8911] [General - Components / Directives / etc] [answered]
Certain types of errors don't appear any more on my console suddenly, example: Below I am adding a component to the components object without importing it. 

`components: {
    Search,
  },`   

and here is what I get: 

![image](https://user-images.githubusercontent.com/24841143/114986533-c8730500-9e9c-11eb-91e8-61b197339329.png)
No errors with a blank screen.

Do you have any idea?

**Comment by metalsadman** [comment, marked as answer]
I can see it in your logs, altho colored yellow, but those are actual errors handled by vue and you should resolve them.

**Reply by saifobeidat** [reply]
oops ! I didn't expect to see errors in yellow! thanks!
